### PR TITLE
table definitions added to the output_definition.json

### PR DESCRIPTION
**Pull request by ottojolanki**

### CODE REVIEW

File: output_definition.json [hunk 1]
```diff
@@ -1,61 +1,78 @@
...
         },
         "star_qc_json": {
-            "path": "star/rep${i+1}/${basename}"
+            "path": "star/rep${i+1}/${basename}",
+            "table": "Star/Replicate ${i+1}/Quality control json"
         },
         "star_qc_log": {
-            "path": "star/rep${i+1}/${basename}"
+            "path": "star/rep${i+1}/${basename}",
+            "table": "Star/Replicate ${i+1}/Python log"
```
Comment (strattan) [inline output_definition.json]: NIT:  Maybe STAR log?

File: output_definition.json [hunk 2]
```diff
@@ -1,61 +1,78 @@
...
-            "path": "wigtobigwig/rep${i+1}/${basename}"
+            "path": "wigtobigwig/rep${i+1}/${basename}",
+            "table": "Wigtobigwig/Replicate ${i+1}/Minus strand bigwig of uniquely mapped reads"
         }
     },
     "mirna_seq_pipeline.spearman_correlation": {
         "spearman_json": {
-            "path": "spearman/${basename}"
+            "path": "spearman/${basename}",
+            "table": "Spearman/Replicates spearman correlation"
```
Comment (strattan) [inline output_definition.json]: NIT:  "Spearman"

File: output_definition.json (hunk 1 above)
Comment (ottojolanki) [reply output_definition.json]: This one is actual python log from the script that produces the quality control json, so the naming is accurate as it is.

File: output_definition.json (hunk 2 above)
Comment (ottojolanki) [reply output_definition.json]: fixed

**Pull request merged**